docs/sources/next/extensions/explore.md

```diff
@@ -287,10 +247,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-sql</h4>
         <p>Load-test SQL Servers</p>
     </a>
-    <a href="https://github.com/mridehalgh/xk6-sqs" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements
https://github.com/mridehalgh/xk6-sqs/issues/8